core-plugins/src/test/java/co/cask/hydrator/plugin/batch/sink/KVTableSinkTest.java

```diff
@@ -50,7 +55,10 @@ public void testTableSinkWithMissingValueField() {
     KVTableSink.KVTableConfig kvTableConfig = new KVTableSink.KVTableConfig("purchases", "rowKey", "user");
     KVTableSink kvTableSink = new KVTableSink(kvTableConfig);
 
-    MockPipelineConfigurer mockPipelineConfigurer = new MockPipelineConfigurer(inputSchema);
+    MockPipelineConfigurer mockPipelineConfigurer = new MockPipelineConfigurer(inputSchema,
+                                                                               ImmutableMap.<String, Object>of(
```

Comment:
I don't imagine these KVTable tests require a validator. Should remove if not needed.